ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -522,13 +524,25 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
         onHeaderViewTapped?()
     }
 
-    private var attributedTitle: NSAttributedString? {
-        didSet {
-            isUsingAttributedTitle = attributedTitle != nil
+    private var attributedTitle: NSAttributedString?
+
+    private var hasOverriddenAttributedForegroundColor: Bool {
+        if let attributes = attributedTitle?.attributes(at: 0, effectiveRange: nil) {
```

Comment:
this is a niche case but in the case that clients only override one letter's foreground color, we preserve that they are responsible for overriding all the other letters as well right?

Reply:
Yep! A screenshot to verify. I applied a green foreground color attribute to just the substring "Footer - A description". The rest is left in the default UITextView text color.

![Screenshot 2024-07-24 at 1 01 14 PM](https://github.com/user-attachments/assets/8c9281e3-d90b-4c28-8165-fd43dd2003e1)